components/led_strip/led_vu.h

```diff
@@ -0,0 +1,29 @@
+/* 
+ *  Control of LED strip within squeezelite-esp32 
+ *     
+ *  (c) Wizmo 2021
+ * 
+ *  This software is released under the MIT License.
+ *  https://opensource.org/licenses/MIT
+ *
+ */
+
+#include <ctype.h>
+
+#define LED_VU_SCALE    100
+
+#define led_vu_color_red()    led_vu_color_all(255, 0, 0)
+#define led_vu_color_green()    led_vu_color_all(0, 255, 0)
+#define led_vu_color_blue()    led_vu_color_all(0, 0, 255)
+#define led_vu_color_yellow()    led_vu_color_all(128, 128, 0)
+
+static struct led_strip_t* led_vu;
```

Comment:
Are you sure this is what you want? static is and odd class for a header file. Don't you want extern instead?

Reply:
Done.  Using like the display structure to conditionally run code